packages/app/src/debug/DebugSpec.vue

```diff
@@ -16,11 +16,64 @@
           class="flex w-full grid px-18px gap-y-8px items-center"
         >
           <div class="flex-grow flex w-full gap-x-2 truncate items-center">
-            <IconDocumentText
-              stroke-color="gray-500"
-              fill-color="gray-100"
-              size="16"
-            />
+            <Tooltip
+              v-if="foundLocally"
+              placement="bottom"
+              color="dark"
+              data-cy="open-in-ide"
+            >
+              <OpenFileInIDE
+                v-slot="{onClick}"
+                :file-path="specData.fullPath"
+              >
+                <button
+                  class="rounded-md border-1px border-gray-100 p-4px group hocus:border-indigo-200"
+                  @click="onClick"
+                >
```

Comment:
Could use an `aria-label` for A11y to say what this button does.

Reply:
Addressed in [dab6639](https://github.com/cypress-io/cypress/pull/25691/commits/dab6639dbf5507643dc69f84109964232cab2229)